test/vite.config.mjs

```diff
@@ -0,0 +1,19 @@
+import { defineConfig } from 'vite';
+
+export default defineConfig(() => {
+	return {
+		server: {
+			headers: {
+				'Cross-Origin-Opener-Policy': 'same-origin',
+				'Cross-Origin-Embedder-Policy': 'require-corp',
+			},
```

Comment:
Do we need these? We're not using Vite server after this PR, right?

Reply:
Ah yeah; removed!